### ISSUE TITLE
about experimental results in the paper

### ISSUE THREAD

**Issue by ZZZ527** [MingshiYan/BIPN#1]
Hi, very good job.
However, when I reproduced it, the taobao data set did not achieve the effect in the paper. The following is my log file.
[taobao_enb_64_2024-10-23 12_17_16.log](https://github.com/user-attachments/files/17500466/taobao_enb_64_2024-10-23.12_17_16.log)

**Comment by MingshiYan** [comment]
Thank you for following my work. You can try setting the learning rate to lr=0.0001. If you're still unable to reproduce the results of the paper, consider performing a grid search on the hyperparameters (learning rate and beta).

**Comment by ZZZ527** [comment]
Thanks for your reply. I tried setting lr to 0.0001 according to the parameter configuration in the log file, but the performance did not achieve the description in the paper. In addition, I don’t understand what the check_point file is used for. Looking forward to your reply.

**Comment by MingshiYan** [comment]
The checkpoint file is not applicable here because if_load_model=False.  Some parameters are redundant, as I didn't remove them from the code.

**Comment by ZZZ527** [comment]
OK, thank you for your reply,it really helped me a lot. I will close this issue and give you a star.
